config/routes.rb

```diff
@@ -70,7 +70,7 @@
     resource :hca_attachments, only: :create
 
     # Excluding this feature until external service (CARMA) is connected
-    resources :caregivers_assistance_claims, only: :create
+    resources :caregivers_assistance_claims, only: :create unless Rails.env.production?
```

Comment:
Just so I'm clear, the CARMA functionality hasn't launched in production, right?

Reply:
Yes, that's correct.